### PR TITLE
Elcomsoft.BlackberryBackup version 10.02.13.6780

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102163)

### CODE REVIEW

File: manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.ElcomsoftBlackberryBackup
```
Comment (Trenly) [inline manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.installer.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.BlackberryBackup
```

File: manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.ElcomsoftBlackberryBackup
+PackageVersion: 10.02.13.6780
+PackageLocale: en-US
+Publisher: Elcomsoft Co. Ltd.
+PackageName: Elcomsoft Blackberry Backup Explorer
+License: Trial
```
Comment (Trenly) [inline manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.locale.en-US.yaml:9]:
```suggestion
License: Trialware
```

File: manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.ElcomsoftBlackberryBackup
```
Comment (Trenly) [inline manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.BlackberryBackup
```

File: manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Elcomsoft.ElcomsoftBlackberryBackup
```
Comment (Trenly) [inline manifests/e/Elcomsoft/ElcomsoftBlackberryBackup/10.02.13.6780/Elcomsoft.ElcomsoftBlackberryBackup.yaml:4]:
```suggestion
PackageIdentifier: Elcomsoft.BlackberryBackup
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
> 'manifests\e\Elcomsoft\ElcomsoftBlackberryBackup\10.02.13.6780', expected 
> 'manifests\e\Elcomsoft\BlackberryBackup\10.02.13.6780'

There's an extra `Elcomsoft` in the path, before `BlackberryBackup`.

**Comment by upintheairsheep** [comment]
I really wish somebody with GIT desktop could save these pull requests for me.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**